rpcs3/rpcs3qt/pad_led_settings_dialog.ui

```diff
@@ -0,0 +1,207 @@
+        </item>
+        <item>
+         <widget class="QPushButton" name="b_colorpicker">
+          <property name="text">
+           <string>Select color...</string>
+          </property>
+         </widget>
+        </item>
+       </layout>
+      </widget>
+     </widget>
+    </item>
+    <item>
+     <widget class="QGroupBox" name="gb_battery_status">
+      <property name="minimumSize">
+       <size>
+        <width>0</width>
+        <height>75</height>
+       </size>
+      </property>
+      <property name="title">
+       <string>Battery status</string>
+      </property>
+      <widget class="QWidget" name="verticalLayoutWidget_2">
+       <property name="geometry">
+        <rect>
+         <x>10</x>
+         <y>20</y>
+         <width>251</width>
+         <height>51</height>
+        </rect>
+       </property>
+       <layout class="QVBoxLayout" name="verticalLayout_2">
+        <item>
+         <widget class="QCheckBox" name="cb_led_blink">
+          <property name="text">
+           <string>Blink LED when battery is low (in-game)</string>
+          </property>
+         </widget>
+        </item>
+        <item>
+         <widget class="QCheckBox" name="cb_led_indicate">
+          <property name="text">
+           <string>Use LED as a battery indicator</string>
+          </property>
+         </widget>
+        </item>
+       </layout>
+      </widget>
+     </widget>
+    </item>
+    <item>
+     <widget class="QGroupBox" name="gb_indicator_brightness">
+      <property name="title">
+       <string>LED battery indicator brightness</string>
+      </property>
+      <widget class="QWidget" name="layoutWidget">
+       <property name="geometry">
+        <rect>
+         <x>10</x>
+         <y>20</y>
+         <width>251</width>
+         <height>33</height>
+        </rect>
+       </property>
+       <layout class="QHBoxLayout" name="lo_brightness" stretch="2,1,13">
+        <item>
+         <widget class="QLabel" name="l_indicator_brightness">
+          <property name="text">
+           <string>100%</string>
+          </property>
+         </widget>
+        </item>
+        <item>
+         <spacer name="horizontalSpacer">
+          <property name="orientation">
+           <enum>Qt::Horizontal</enum>
+          </property>
+          <property name="sizeHint" stdset="0">
+           <size>
+            <width>20</width>
+            <height>20</height>
+           </size>
+          </property>
+         </spacer>
+        </item>
+        <item>
+         <widget class="QSlider" name="hs_indicator_brightness">
+          <property name="maximum">
+           <number>100</number>
+          </property>
+          <property name="orientation">
+           <enum>Qt::Horizontal</enum>
+          </property>
+         </widget>
+        </item>
+       </layout>
+      </widget>
+     </widget>
+    </item>
+   </layout>
+  </widget>
+  <widget class="QWidget" name="horizontalLayoutWidget_3">
```

Comment:
you should replace all of the button layout stuff with a Dialog Button Box.
You then have to change the connect for the buttons as seen here for example:
https://github.com/RPCS3/rpcs3/blob/9084209cfcc42e115fedb27b5f5da0089bfc8c94/rpcs3/rpcs3qt/pkg_install_dialog.cpp#L64-L78

Reply:
Done in c34910b, I opted for OK / Cancel / Apply. To support resetting the values back (Cancel) after applying them, I moved most member variables into a struct so as to make the creation of its copy in the constructor graceful.